src/model_config_tests/qa/test_access_esm1p5_config.py

```diff
@@ -181,6 +181,10 @@ def test_mppnccombine_fast_collate_exe(self, config):
                 "collate.exe", "config.yaml", VALID_MPPNCCOMBINE_EXE
             )
 
+            assert "mpi" in config["collate"], error_field_nonexistence(
```

Comment:
`codecov` in GitHub is warning about this missing test coverage. I this something required for the PR?

Reply:
This would require adding tests for tests, which I don't think we have time for at the moement. It's on the list, though: https://github.com/ACCESS-NRI/model-config-tests/issues/44

Reply:
Ok, that sounds like an endless self referencing problem...